kgforge/core/forge.py

```diff
@@ -224,13 +224,47 @@ def validate(self, data: Union[Resource, List[Resource]],
 
     # Resolving User Interface.
 
-    def resolvers(self):
-        print("Available scopes:")
-        for k, v in sorted(self._resolvers.items()):
-            print(" - ", k, ":")
-            for r_key, r_value in v.items():
-                print("     - resolver: ", r_key)
-                print("         - targets: ",  ",".join(r_value.targets.keys()))
+    def resolvers(self, as_dict: bool = False) -> Optional[Dict]:
+        """
+        Prints or returns the available resolvers of the graph.
+
+        If as dict is False (default), prints the available scopes
+        If as_dict is True, returns a dictionary of the available scores and their targets
```

Comment:
Minor typo: 'scopes' here

Reply:
I changed the pydoc and pushed the changes